.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,92 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: 'Run tests with php ${{ matrix.php-version }}'
+        runs-on: ubuntu-latest
+
+        env:
+            SYMFONY__DATABASE__DRIVER: pdo_mysql
+            SYMFONY__DATABASE__VERSION: 5.7
+            SYMFONY__DATABASE__HOST: 127.0.0.1
+            SYMFONY__DATABASE__PORT: 3306
+            SYMFONY__DATABASE__NAME: sulu_test
+            SYMFONY__DATABASE__USER: root
+            SYMFONY__DATABASE__PASSWORD: root
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '5.5'
+                      composer-flags: '--prefer-dist --no-interaction'
+                      tools: 'composer:v1'
+                      lint: false
+                      env:
+                          SYMFONY_DEPRECATIONS_HELPER: disabled
+
+                    - php-version: '7.2'
```

Comment:
Unfortunately that's not possible because some library is using `continue;` in a `switch` statement and therefore this error (https://github.com/luca-rath/SuluCommentBundle/runs/1696707179) happens in php 7.3 and above